src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -3038,6 +3121,11 @@ object Desugar {
     def typ     (n: String, context: ExternalTypeInfo): String = nameWithoutScope(TYPE_PREFIX)(n, context)
     def field   (n: String, @unused s: StructT): String = s"$n$FIELD_PREFIX" // Field names must preserve their equality from the Go level
     def function(n: String, context: ExternalTypeInfo): String = nameWithoutScope(FUNCTION_PREFIX)(n, context)
+    def outlinedFunction(context: ExternalTypeInfo) = {
+      val name = nameWithoutScope(FUNCTION_PREFIX)("outline_"+outlineCounter, context)
+      outlineCounter += 1
+      name
+    }
```

Comment:
add the return type. Also, try that the name does not depend on some counter. e.g. you could use the location information.

Reply:
I decided to use the counter because there can be an arbitrary number of outline blocks in a given function. What location information could I use?